taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -87,19 +87,18 @@ def __init__(self, projectid, sampleid, **kwargs):
         self.reportpath = getattr(self, 'reportpath', None)
         self.force = getattr(self, 'force', False)
         self.stage_only = getattr(self, 'stage_only', False)
+        self.projectname = None
         #Fetches a project name, should always be availble; but is not a requirement
         try:
-            n = db.project_entry(db.dbcon(), projectid)['name']
-            self.projectname = n
+            self.projectname = getattr(db.project_entry(db.dbcon(), projectid), 'name')
         except KeyError:
```

Comment:
As you probably noticed, the travis build fails because `getattr` throws an `AttributeError` and not a `KeyError`